packages/vapor/scss/variables.scss

```diff
@@ -554,6 +553,7 @@ $logo-card-padding: 1rem;
 $logo-card-margin: 0.25rem;
 $logo-card-disabled-opacity: 0.75;
 $logo-card-disabled-filter: grayscale(100%);
+$logo-card-border-radius: 8px;
```

Comment:
Is this used anywhere?

Reply:
No, I think I might have re-added that, thanks! I will remove :)